src/crimson/os/seastore/journal/circular_bounded_journal.cc

```diff
@@ -31,23 +28,19 @@ CircularBoundedJournal::CircularBoundedJournal(
     const std::string &path)
   : trimmer(trimmer), device(device), path(path) {}
 
-CircularBoundedJournal::mkfs_ret
-CircularBoundedJournal::mkfs(const mkfs_config_t& config)
+CircularBoundedJournal::mkfs_ret CircularBoundedJournal::mkfs()
```

Comment:
To work with `TransactionManager::mkfs()`, merge into `open_for_mkfs()`.

Reply:
Done.